src/index.pug

```diff
@@ -0,0 +1,18 @@
+doctype html
+html
+    head
+        meta(name="viewport" content="width=device-width")
+        meta(name="description" content="This is the version 2 of My-cv for junior and we gonna use pug as templete engine for hrml")
+        title Welcome to My CV Version 2 Junior
+        //- link(rel="stylesheet", href="./style.scss")
+        if config.entry.css.embed
+        link(rel="stylesheet" href=`${embedPath}` embed)
+    body    
+        .content
+            include _component/Welcome/Welcome 
```

Comment:
WHY OH WHY do you use capital letters in naming folders and files?
don't become @CarolEnache

Reply:
![image](https://user-images.githubusercontent.com/44984647/71131984-2ed70d00-21ee-11ea-8afd-1142bebca497.png)
I'm using normal writing, probably the autogenerate for the path does it.

Reply:
well, no more not paying attention then, we're coders here, not casual text chatters